### ISSUE TITLE
<c-c> leaves insert mode even if control-keys are disabled

### ISSUE THREAD

**Issue by t-animal** [asvetliakov/vscode-neovim#243]
Expected behaviour: With ctrl keys disabled in insert mode, Ctrl-C should copy selected text normally.

Actual behaviour: Ctrl-C leaves insert mode without copying anything

Steps to reproduce: 
1) Install and setup extension
2) Disable control-keys in insert mode ("vscode-neovim.useCtrlKeysForInsertMode": false)
3) Restart VSCode
4) In a document, enter insert mode and select some text
5) Press Ctrl-C

**Comment by t-animal** [comment]
I even tried remapping <C-c> in ~/.config/nvim/init.vim:
```
imap <C-c> <nop>
```
This actually disables leaving the insert mode on Ctrl-C in neovim, but not in the vscode-neovim extension :-(

**Comment by t-animal** [comment]
I have found a workaround. To avoid this behaviour I put 
```  
{
    "command": "-vscode-neovim.escape",
    "key": "ctrl+c"
  }
```
in my `keybindings.json`

**Comment by KevinNThomas** [comment]
Thanks a lot for this fix, worked for me as well.

**Comment by theol0403** [comment]
Fixed by #513
